### PR TITLE
Javscript classes

### PR DESCRIPTION
**Pull request by SaneMyburg**
The following is a list of completed tasks:

- Refactoring Javascript object and arrays to use classes.
- Add styling to the display section.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,13 +1,16 @@
 /* eslint-disable max-classes-per-file */
```
Comment (t-yanick) [inline index.js:1]: - [ ] Great job working on this milestone. You are not suppose to disable eslint, rather fix the errors resulting from running the linter checks.

File: index.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
 // Store Class: Handles local storage of books
-class Storage {/* eslint-disable max-classes-per-file */
+class Storage {
+  /* eslint-disable max-classes-per-file */
```
Comment (t-yanick) [inline index.js]: - [ ] Great job working on this milestone. You are not suppose to disable eslint, rather fix the errors resulting from running the linter checks.

File: index.js [hunk 3]
```diff
@@ -44,7 +47,8 @@ class Storage {/* eslint-disable max-classes-per-file */
 
 // UI Class: Displays listed Books
 
-class UI {/* eslint-disable max-classes-per-file */
+class UI {
+  /* eslint-disable max-classes-per-file */
```
Comment (t-yanick) [inline index.js]: - [ ] Great job working on this milestone. You are not suppose to disable eslint, rather fix the errors resulting from running the linter checks.

File: index.js [hunk 4]
```diff
@@ -93,7 +96,9 @@ document.querySelector('#book-collection').addEventListener('click', (e) => {
...
   // Remove book from Store
-  Storage.removeBook(e.target.previousElementSibling.previousElementSibling.textContent);
+  Storage.removeBook(
+    e.target.previousElementSibling.previousElementSibling.textContent,
+  );
 });
 
-/* eslint-disable max-classes-per-file */
\ No newline at end of file
+/* eslint-disable max-classes-per-file */
```
Comment (t-yanick) [inline index.js:102]: - [ ] Great job working on this milestone. You are not suppose to disable eslint, rather fix the errors resulting from running the linter checks.

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @SaneMyburg and @Leeoasis , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Classes used instead of objects and array
:heavy_check_mark: Plain JavaScript used, no framework
:heavy_check_mark: All linter checks passing

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Leeoasis) [reply index.js:1]: hie @t-yanick  What we did was advised in the project requirements hence we are a little bit confused, could you please shed some more light on how to do that?

**Review by ShadyShawkat — APPROVED** [review]
## Hi @SaneMyburg, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**